docs/Makefile

```diff
@@ -6,6 +6,7 @@ SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = build
+DOTCMD        = .buildfigures.sh
```

Comment:
Change this to `./.buildfigures.sh` for those of us that haven't added `.` to our path.